datasets/src/lib.rs

```diff
@@ -58,6 +67,20 @@ fn array_from_buf(buf: &[u8]) -> Array2<f64> {
     reader.deserialize_array2_dynamic().unwrap()
 }
 
+#[cfg(feature = "income")]
+fn dataframe_from_buf(buf: &[u8]) -> DataFrame {
+    let mut file = GzDecoder::new(buf);
+    let mut buf: Vec<u8> = Vec::new();
+    file.read_to_end(&mut buf).unwrap();
+    let buf = Cursor::new(buf);
+
+    CsvReader::new(buf)
+        .infer_schema(None)
+        .has_header(false)
```

Comment:
Shouldn't this option be give as a function argument `has_header: bool`?

Reply:
Hi, Is not it better to have direct arrow support than providing support for polars. So that any dataframe library based on arrow2 can use the preprocessing, and training methods? I am new to Rust and lifa project my question could be stupid. Thanks in advance for any clarity.